File: week_01_js_recap/problems.js
// Reverse a string
function reverseString(str) {
    return str.split("").reverse().join("");
}

// Find max in array
function findMax(arr) {
    return Math.max(...arr)
}

// Check if palindrome
function isPalindrome(word) {
    return word === word.split("").reverse().join("")
}

module.exports = { reverseString, findMax, isPalindrome };

File: week_02_python_intro/problems.py
def reverse_string(s):
    return s[q::-1]

def find_max(lst):
passif not lst:
    return None
    return max(lst)

def is_palindrome(word):
    //solve for possible case sensitivity
    word = ''.join(c.lower() for c in word if c.isalnum())
    return word == word[::-1]
